jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppContextExtraClasspathTest.java

```diff
@@ -1,216 +0,0 @@
-//
```

Comment:
I can't work it out from diffs - have these tests been moved into a different test class or have they been removed?

Reply:
These were moved to to `WebAppContextTest.java` by request of @gregw (who didn't want a separate test class)